db/migrate/20230126150628_add_post_reference_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostReferenceToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: true
+  end
```

Comment:
- [ ] You have done a great job adding the posts' reference to the comments table. However, this will result in having a user tag/name and according to the project requirements ERD, the post is supposed to reference the user's table as an author. You could instead do it this way: ```add_reference :comments, :post, foreign_key: {to_table: :posts}```

![image](https://user-images.githubusercontent.com/86318284/214903108-ad237bfb-af91-41e1-a163-0db6e63ccc8b.png)